src/tests/serialization.rs

```diff
@@ -67,16 +65,18 @@ fn server_registration_roundtrip() {
     assert_eq!(reg_bytes, oprf_bytes);
     // If we do have envelope and client pk, the server registration contains
     // the whole kit
-    let rkr_size = RKRCiphertext::<ChaCha20Poly1305>::rkr_with_nonce_size();
-    let mut mock_rkr_bytes = vec![0u8; rkr_size];
-    rng.fill_bytes(&mut mock_rkr_bytes);
-    println!("{}", mock_rkr_bytes.len());
+    let key_len =
+        <<<Default as CipherSuite>::KeyFormat as KeyPair>::Repr as SizedBytes>::Len::to_usize();
```

Comment:
not: I suspect you do not need all the `as Trait` here, and there's a higher probability that the inner one is not required.

Reply:
I tried removing each "as Trait", and each one causes the same error to appear.